scripts/preprocess_spec.py

```diff
@@ -30,24 +39,67 @@ def _to_camel_case(s):
     return ''.join(_title(y) for y in s.split("_"))
 
 
-def iterate_through_operations(spec, func):
+def apply_funcs_to_spec(spec, func, *params):
+    """Apply fun to each operation in the spec.
+
+    First two parameters of func will be operation and it's parent, and the
```

Comment:
(No action required)

Consider defining the signature rather than explaining it (e.g. ``func(operation, parent)``).